int32 indexCategory
int32 indexChannel
string description
int32 layer
float32 vigilance
float32 learningRate